invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/detail.html

```diff
@@ -274,11 +275,19 @@
               </section>
             {%- endblock record_title -%}
 
+            {%- block after_record_title -%}
+              {# After record title hook #}
+            {%- endblock after_record_title -%}
```

Comment:
minor/question: wouldn't it also work to just override `record_title` and added after?

```jnja
{% block record_title %}

{# {# Things here are the same as a "before_record_title" block #} #}

{{ super() }}

{# Things here are the same as a "after_record_title" block #}
{% endblock record_title %}
```

My concern is that we already lack a solid "Blocks API" for our templates, and adding more names/IDs might mean a rougher path to standardisation.